client/src/components/Sub_RelatedItems/SingleProd.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+import ProdImg from './Sub_SingleProd/ProdImg.jsx';
+import ProdInfo from './Sub_SingleProd/ProdInfo.jsx';
+
+function SingleProd({ product, setProductID, productStyle, reviewMetadata, onClick }) {
+
+  // =================== STATES ===================
+
+  const [theseStyles, setTheseStyles] = useState([]);
+
+  const [thisPrice, setThisPrice] = useState([]);
+
+  const [thisAvgRating, setThisAvgRating] = useState([]);
+
+  const [imgUrl, setImgUrl] = useState('');
+
+  // =================== EFFECTS ===================
+
+  useEffect(() => {
+    axios.get(`/api/products/${id}/styles`)
+      .then((styles) => {
+        const newStyles = styles.data.results;
+        setTheseStyles(newStyles);
+        setImgUrl('');
+        setImgUrl(newStyles[0].photos[0].url);
```

Comment:
1) It appears that you display the first photo of the first style. I think it would be more appropriate to use the first photo of the default style instead. because you display the price of the default style. it would be better if the price and the photo is matched
2) what if the url is empty?

Reply:
I am glad you brought up the photo style issue, when reading the BRD it was not clear to me which photo is intended as the display. Maybe we can discuss for a few minutes after this morning's standup?